cpp/test/Ice/optional/AllTests.cpp

```diff
@@ -903,50 +903,6 @@ allTests(Test::TestHelper* helper, bool)
 
     if(communicator->getProperties()->getPropertyAsInt("Ice.Default.SlicedFormat") > 0)
     {
-        cout << "testing marshalling with unknown class slices... " << flush;
```

Comment:
It's not immediately clear why you removed this test.

Reply:
I removed it because it didn't seem very useful to test slicing here. But on second thoughts, it's indeed useful to ensure the "has optional members" information isn't lost for the unknown slice. I've added it back.